src/client/MapleCharacter.java

```diff
@@ -1695,15 +1692,16 @@ public void saveToDB(boolean dc, boolean fromcs) {
                 buddylist.setChanged(false);
             }
 
-            ps = con.prepareStatement("UPDATE accounts SET `nxCredit` = ?, `ACash` = ?, `mPoints` = ?, `points` = ?, `vpoints` = ?, `dpoints` = ?, `epoints` = ? WHERE id = ?");
+            ps = con.prepareStatement("UPDATE accounts SET `nxCredit` = ?, `ACash` = ?, `mPoints` = ?, `rPoints` = ?, `points` = ?, `vpoints` = ?, `dpoints` = ?, `epoints` = ? WHERE id = ?");
```

Comment:
Using backticks is only required for reserved keywords (and using reserved keywords is bad programming to begin with).